vllm_ascend/compilation/compiler_interface.py

```diff
@@ -39,6 +42,70 @@ def compile_fx(graph: GraphModule, example_inputs: list,
     return aot_autograd(fw_compiler=inner_compile)(graph, example_inputs)
 
 
+def fusion_pass_compile(
+    graph: fx.GraphModule,
+    example_inputs: list[Any],
+    compiler_config: dict[str, Any],
+    runtime_shape: Optional[int] = None,
+    key: Optional[str] = None,
+) -> tuple[Optional[Callable], Optional[Any]]:
+
+    def compile_inner(graph, example_inputs):
+        current_pass_manager = compiler_config["graph_fusion_manager"]
+        graph = current_pass_manager(graph, runtime_shape)
+        return graph
+
+    decompositions = select_decomp_table()
+
+    compiled_fn = compile_fx(
+        graph=graph,
+        example_inputs=example_inputs,
+        inner_compile=compile_inner,
+        decompositions=decompositions,
+    )
+
+    return compiled_fn, None
+
+
+def npugraph_ex_compile(
+    graph: fx.GraphModule,
+    example_inputs: list[Any],
+    compiler_config: dict[str, Any],
+    runtime_shape: Optional[int] = None,
+    key: Optional[str] = None,
+) -> tuple[Optional[Callable], Optional[Any]]:
+    # When currently using the FULL_DECODE_ONLY mode,
+    # the piecewise compilation level slicing process
+    # in vllm is also encountered.
+    # This process causes the output to no longer be
+    # wrapped as a tuple when the fx graph has a single
+    # output, but torch.compile has a mandatory check.
+    fx_graph = graph.graph
+    if not graph_returns_tuple(graph):
+        output_node = fx_graph.output_node()
+        with fx_graph.inserting_before(output_node):
+            return_value = output_node.args[0]
+            tuple_node = fx_graph.create_node("call_function",
+                                              tuple,
+                                              args=([return_value], ))
+        output_node.args = (tuple_node, )
+        fx_graph.recompile()
+
+    import torchair
+
+    import vllm_ascend.compilation.npugraph_ex_passes.add_rms_norm_quant  # noqa
```

Comment:
Please add note for this: Why do we need explicit imports? What's the magic will be happen here?